services/repology/repology-repositories.spec.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const { test, given } = require('sazerac')
+const RepologyRepositories = require('./repology-repositories.service')
+
+describe('Repology repositories count badge', function() {
+  test(RepologyRepositories.render, () => {
+    given({ repositoryCount: 0 }).expect({ message: 0, color: 'blue' })
+    given({ repositoryCount: 18 }).expect({ message: 18, color: 'blue' })
+  })
+})
```

Comment:
We can remove these altogether. For the classes that integrate with the upstream services/platforms, we typically only add unit/function level tests like these for transformation (parsing/extraction) and rendering logic which we fortunately don't have here!